ja-jp/content/topics/apps/office365/advanced.htm

```diff
@@ -122,7 +122,7 @@
                                 </div>
                             </div>
                             <div class="title-bar-layout outer-row">
-                                <div class="logo-wrapper"><a class="logo" href="https://support.okta.com/help/s/?language=ja" alt="Okta"></a>
+                                <div class="logo-wrapper"><a class="logo" href="https://help.okta.com/okta_help.htm?id=csh-index" alt="Okta"></a>
```

Comment:
same question as in the OIE PR, does the alias redirect for `csh-index` goes to Japanese content?